chat-commands.js

```diff
@@ -2452,18 +2449,12 @@ const commands = {
 		if (targetUser && (cmd === 'hidealtstext' || cmd === 'hidetextalts' || cmd === 'hidealttext')) {
 			this.addModAction(`${name}'s alts' messages were cleared from ${room.title} by ${user.name}.`);
 			this.modlog('HIDEALTSTEXT', targetUser, null, {noip: 1});
-			this.add(`|unlink|hide|${userid}`);
-			const alts = targetUser.getAltUsers(true);
-			for (const alt of alts) {
-				this.add(`|unlink|hide|${alt.getLastId()}`);
-			}
-			for (const prevName in targetUser.prevNames) {
-				this.add(`|unlink|hide|${targetUser.prevNames[prevName]}`);
-			}
+			const alts = [userid].concat(Object.keys(targetUser.prevNames)).concat(targetUser.getAltUsers(true).map(user => user.getLastId()));
+			room.hideText(alts);
 		} else {
 			this.addModAction(`${name}'s messages were cleared from ${room.title} by ${user.name}.`);
```

Comment:
Like the other `send`, this should also be `sendModAction` now.

Reply:
(Although check to see if `sendModAction` even exists; you may need to manually implement the corresponding functionality.)